features/filesystem/fat/FATFileSystem.cpp

```diff
@@ -160,6 +160,26 @@ void ff_memfree(void *p)
 }
 
 // Implementation of diskio functions (see ChaN/diskio.h)
+static WORD disk_get_sector_size(BYTE pdrv)
+{
+    WORD ssize = _ffs[pdrv]->get_erase_size();
```

Comment:
It probably makes sense to use `get_read_size` here (the invariant here is that read size == wirte size, right?) since `get_erase_size()` can assume whatever value the HW reports and the block driver will act accordingly.
The rounding to 512 isn't needed anymore since we found the erase size for that particular card isn't 128 but 64K, I think.